tests/src/test/scala/com/nvidia/spark/rapids/StringFunctionSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+class StringFunctionSuite extends SparkQueryCompareTestSuite {
+  testSparkResultsAreEqual("Test repeat string(s) with scalar string and column repeatTimes",
+    intsDf) {
+    frame => frame.selectExpr(
+      "repeat(NULL, ints)",
```

Comment:
Does this actually test anything?  This will be similar to the Python test that passed a NULL literal, and I would expect Spark's optimizer to simplify this to a simple `null` project.  Adding tests for `NULL` inputs will similarly require bypassing Spark like the testing for scalars above.

Reply:
Yeah, I have no idea why. My code actually ran that test (I just throw an exception in my function, and this case invokes such throwing in doColumnar(scalar-column)).

Reply:
If you are just checking if doColumnar(scalar,column) is being called then that will be true due to the second expression being used, `repeat('abc123', ints)` which won't be optimized out.

Reply:
No, I tested that for one expression at a time. Yeah, consistently inconsistent :smiley: